### PR TITLE
Playwright e2e tests

### PR DESCRIPTION
**Pull request by oznasi1**

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
```
Comment (zavelevsky) [inline .github/workflows/playwright.yml:4]: we don't have master branch - why keep it?

File: e2e/auth.setup.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { test as setup } from '@playwright/test';
+
```
Comment (zavelevsky) [inline e2e/auth.setup.ts:2]: test files should be named accordingly e.g. .spec.ts or test.ts

File: src/libs/modals/Modal.tsx [hunk 3]
```diff
@@ -49,6 +49,7 @@ export const Modal: FC<Props> = ({
       className={'px-content items-center justify-center'}
     >
       <m.div
+        id="modal"
```
Comment (zavelevsky) [inline src/libs/modals/Modal.tsx:52]: WAT

**Comment by oznasi1** [comment]
Check

**Pull request closed without merging**